### PR TITLE
SK-1671-TRANS-WEB-Emails-MXplan-Zimbra-Webmail-Calendar_feature

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.es-es.md [hunk 1]
```diff
@@ -360,6 +388,72 @@ Seleccione una de las dos fichas siguientes:
 >> ![Zimbra - Exporter](images/zimbra-20.png){.thumbnail}
 >>
 
+### Calendario <a name="calendario"></a>
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.es-es.md]: anchor is wrong

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.es-es.md [hunk 2]
```diff
@@ -289,6 +292,31 @@ Su interfaz Zimbra dispone de 2 menús de configuración:
 >>
 >> Para configurar los filtros, consulte la sección "[Filtros](#filters)" de esta documentación.
 >>
+> **Agenda y recordatorios**
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.es-es.md:295]: is it Agenda or Calendario in Zimbra ? If possible, I'd recommend to have a common term everywhere

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.it-it.md [hunk 3]
```diff
@@ -360,6 +388,72 @@ Selezionare una delle due schede seguenti:
...
+- **Mostra come**: definisci se l'evento rende disponibili o meno i suoi ospiti durante lo svolgimento
+- **Agenda**: definire il calendario associato all'evento.
+
+Una volta definito l’evento, clicca su `Registra`{.action}.
+
+![Zimbra - calendar](images/zimbra-calendar-event-add-02.png){.thumbnail .w-400}
+
+##### 3- Modificare un evento <a name="calendar-modify-event"></a>
+
+#### Task <a name="tasks"></a>
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.it-it.md]: Not translated ? (same in the summary)

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.pl-pl.md [hunk 4]
```diff
@@ -54,6 +54,9 @@ OVHcloud dostarcza usługę webmail o nazwie Zimbra umożliwiającą dostęp do
     - [Zarządzanie folderami](#contacts-folders)
     - [Zarządzanie listami](#contacts-lists)
     - [Import / Eksport kontaktów](#import-export)
+- [Kalendarz](#calendar)
+    - [Zarządzanie kalendarzami](#calendar-management)
+    - [Zadania](#zadania)
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.pl-pl.md]: anchor is translated

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.pl-pl.md [hunk 5]
```diff
@@ -360,6 +388,72 @@ Wybierz jedną z dwóch następujących zakładek:
 >> ![Zimbra - Eksport](images/zimbra-20.png){.thumbnail}
 >>
 
+### Kalendarz <a name="kalendarz"></a>
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.pl-pl.md]: wrong anchor

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.pt-pt.md [hunk 6]
```diff
@@ -289,6 +292,31 @@ A sua interface Zimbra dispõe de 2 menus de configuração:
 >>
 >> Para configurar os filtros, consulte a secção "[Filtros](#filters)" deste manual.
 >>
+> **calendário e lembretes**
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_zimbra/guide.pt-pt.md]:
```suggestion
> **Calendário e lembretes**
```

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**